tests/globalkilltest/run-tests.sh

```diff
@@ -55,6 +55,26 @@ function help_message()
 '
 }
 
+function start_cluster()
+{
+   ${PD} --name=pd --data-dir=pd &>pd.log &
```

Comment:
Delete `--pd <pd-client-path>: PD client path, ip:port list seperated by comma. Defaults to "127.0.0.1:2379".` in help message.